docs/source/guide.rst

```diff
@@ -284,14 +284,60 @@ Then we can check if our result is correct. ``result.current`` is a value return
 Transactions
 ************
 
-Sending
-=======
+Sending transaction
+===================
+
+Example is available `here <https://example.usedapp.io/send>`_.
+
+Sending transactions is really simple with useDApp. All we need to send a simple transaction,
+is to use :ref:`useSendTransaction` hook, which returns a ``sendTransaction`` function and ``state`` object.
+
+**Example**
+
+Simply call a hook in a component.
+
+.. code-block:: javascript  
+
+  const { sendTransaction, state } = useSendTransaction()
+
+Then when you want to send a transaction, call ``sendTransaction`` for example in a button callback.
+Function accepts a `Transaction Request <https://docs.ethers.io/v5/api/providers/types/#providers-TransactionRequest>`_ object as a parameter.
+
+.. code-block:: javascript  
 
-Example is available `here <https://example.usedapp.io/transactions>`_.
+  const handleClick = () => {
+    setDisabled(true)
+    sendTransaction({ to: address, value: utils.parseEther(amount) })
+  }
```

Comment:
There should be definition of setDisabled in the example otherwise it is hard to understand. Also a note (i.e. "It is a good practice to disable component when transaction is mining").